mmdet/models/layers/msdeformattn_pixel_decoder.py

```diff
@@ -161,9 +138,9 @@ def init_weights(self) -> None:
 
         # init_weights defined in MultiScaleDeformableAttention
         for layer in self.encoder.layers:
-            for attn in layer.attentions:
-                if isinstance(attn, MultiScaleDeformableAttention):
-                    attn.init_weights()
+            attn = layer.self_attn
```

Comment:
I recommend using
```
 for m in self.encoder.layers.modules():
     if isinstance(attn, MultiScaleDeformableAttention):
         m.init_weights()
```

Reply:
This can keep consistent with the original implementation even there is multiple MSDA

Reply:
okkkkk @jshilong